lamindb/_dataset.py

```diff
@@ -210,9 +210,10 @@ def from_df(
     reference_type: Optional[str] = None,
     version: Optional[str] = None,
     is_new_version_of: Optional["File"] = None,
+    **kwargs,
```

Comment:
Why would you like to introduce `**kwargs`?

Reply:
Because I want to pass organism=